docs/changelog.rst

```diff
@@ -30,6 +30,33 @@ Changelog
   - `Maps.me Token (MAPS) <https://www.coingecko.com/en/coins/maps>`__
   - `xToken (XTK) <https://www.coingecko.com/en/coins/xtoken>`__
   - `Mobile Coin (MOB) <https://www.coingecko.com/en/coins/mobilecoin>`__
+  - `AC Milan Fan Token (ACM) <https://www.coingecko.com/en/coins/ac-milan-fan-token>`__
```

Comment:
I just realized that you added all of those tokens in the wrong release @vnavascues .